core/src/main/java/com/microsoft/applicationinsights/internal/config/TelemetryConfigurationFactory.java

```diff
@@ -193,8 +216,12 @@ private void setTelemetryProcessors(ApplicationInsightsXmlConfiguration appConfi
         if (configurationProcessors != null) {
             ArrayList<TelemetryProcessorXmlElement> b = configurationProcessors.getBuiltInTelemetryProcessors();
             if (!b.isEmpty()) {
-                final List<String> processorsBuiltInNames =
-                        new AnnotationPackageScanner().scanForClassAnnotations(new Class[]{BuiltInProcessor.class}, performanceCountersSection);
+                List<String> processorsBuiltInNames = new ArrayList<>();
+                if (System.getProperty(BUILTIN_PROCESSORS_SCANNING_ENABLED_PROPERTY, "false").equalsIgnoreCase("true")) {
```

Comment:
Please add a comment about this property too. It is hard to remember later on.

Reply:
sure